docker/resource_docker_plugin_funcs.go

```diff
@@ -0,0 +1,60 @@
+package docker
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/docker/api/types"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceDockerPluginCreate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ProviderConfig).DockerClient
+	ctx := context.Background()
+	pluginName := d.Get("name").(string)
+	// TODO
+	if err := client.PluginCreate(ctx, nil, types.PluginCreateOptions{
```

Comment:
https://pkg.go.dev/github.com/docker/docker/client#Client.PluginCreate

I don't know the content of `createContext`.

Reply:
* PluginInstall
* PluginCreate